<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="removePartyCancelPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peru</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="removePartyComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Poistettava seurue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removePartyRemovePushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Poista</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>removePartyComboBox</tabstop>
  <tabstop>removePartyCancelPushButton</tabstop>
  <tabstop>removePartyRemovePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
